### PR TITLE
Aerie Native Auth Token Changes

### PR DESCRIPTION
**Pull request by rohan-r-kulkarni**
Changes in aerie_host.py for #17 
- check for error before extracting data from the JSON as to not cause another tradeback
- create an instance variable self.token which stores the token returned for the session with the prefix "Bearer "
- pass in the header {"Authorizaton": self.token} for any GraphQL request made through the CLI.

### CODE REVIEW

**Review by cartermak — COMMENTED** [review]
Thanks for getting this working! In order to merge this, we need to maintain functionality for Aerie < 1.7.0; could you please add in a conditional branch to where we pull out the token that chooses between the old auth and JWT based on whether the response contains the key `"ssoToken"` vs. `"token"`? 

I also added some comments about how best to handle the token.

File: src/aerie_cli/aerie_host.py [hunk 1]
```diff
@@ -65,12 +68,22 @@ def post_to_graphql(self, query: str, **kwargs) -> Dict:
...
 
             if resp.ok:
-                resp_json = resp.json()["data"]
-                data = next(iter(resp_json.values()))
+                err = None
+                try:
+                    err = resp.json()["errors"]
+                except:
+                    pass
+                if err is not None:
```
[the remark this answers is not in the document]
Comment (rohan-r-kulkarni) [reply src/aerie_cli/aerie_host.py:81]: Checking for "errors" in the keys is definitely the way to go. Thanks, will make the change!

File: src/aerie_cli/aerie_host.py [hunk 2]
```diff
@@ -32,7 +33,8 @@ def __init__(
         session: requests.Session,
         graphql_url: str,
         gateway_url: str,
-        configuration_name: str = None
+        configuration_name: str = None,
+        token: str = None
```
[the remark this answers is not in the document]
Comment (rohan-r-kulkarni) [reply src/aerie_cli/aerie_host.py:37]: Ok, basically we should store the token in `session.headers` and not an instance variable of AerieHost. Got it!

**Comment by rohan-r-kulkarni** [comment]
OK, I've made the backward compatibility changes and tested it with a local test script connecting to localhost and eurcdevaerie venues. Let me know if this works.

File: src/aerie_cli/aerie_host.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (cartermak) [reply src/aerie_cli/aerie_host.py]: Yes, nailed it!

**Review by cartermak — APPROVED** [review]
Auth fix looks great! Thanks for verifying on multiple hosts.

**Pull request merged**